service/deployer/eventer/eventer.go

```diff
@@ -0,0 +1,91 @@
+package eventer
+
+import (
+	"time"
+
+	microerror "github.com/giantswarm/microkit/error"
+	micrologger "github.com/giantswarm/microkit/logger"
+)
+
+// EventerType represents the type of Eventer to configure.
+type EventerType string
+
+// Config represents the configuration used to create an Eventer.
+type Config struct {
+	// Dependencies.
+	Logger micrologger.Logger
+
+	// Settings.
+	Type EventerType
+}
+
+// DefaultConfig provides a default configuration to create a new Eventer
+// service by best effort.
+func DefaultConfig() Config {
+	return Config{
+		// Dependencies.
+		Logger: nil,
+
+		// Settings.
+		Type: StubEventer,
+	}
+}
+
+// New creates a new configured Eventer.
+func New(config Config) (Eventer, error) {
+	// Dependencies.
+	if config.Logger == nil {
+		return nil, microerror.MaskAnyf(invalidConfigError, "logger must not be empty")
+	}
+
+	var newService Eventer
+
+	switch config.Type {
+	case StubEventer:
+		newService = &stubEventer{
+			// Dependencies.
+			logger: config.Logger,
+		}
+	default:
+		return nil, microerror.MaskAnyf(invalidConfigError, "could not find eventer type")
+	}
+
+	return newService, nil
+}
+
+// StubEventer is an Eventer that just pretends it's a real Eventer.
+var StubEventer EventerType = "StubEventer"
+
+// stubEventer is a stub implementation of the Eventer interface.
+type stubEventer struct {
+	// Dependencies.
+	logger micrologger.Logger
+}
+
+func (e *stubEventer) NewDeploymentEvents() ([]DeploymentEvent, time.Duration, error) {
```

Comment:
This actually would be nicer. Then we wouldn't leak any of the `waitTime` foo. Will update.